docs/Upgrading-from-legacy-Theos.md

```diff
@@ -20,20 +18,20 @@ $ git submodule update --init --recursive
 * **Configuration related to your environment, rather than the project, should be in `~/.theosrc`.** If you set variables that are specific to your own setup, such as the location of Xcode/an SDK, device IP, etc., you should avoid placing it in your project makefile, as these types of things can vary between users. Move these to `~/.theosrc`, a makefile that is read and executed during an early stage of Theos’ `common.mk`.
-* **Built packages have been moved to a directory called `packages`.** The intent here is to reduce clutter in the root project directory, and separate build output from the project source.
+* **Built packages have been moved to a directory called `packages`.** The intent here is to reduce clutter in the root project directory and separate build output from the project source.
 * **The `obj` directory has been moved to inside `.theos`.** Again, this is for cleanliness. You may need to enable displaying of hidden files in your operating system/file manager.
 * **The linker flag required for tweaks to run in 32-bit processes on 64-bit devices running iOS 9 is not necessary.** Theos already applies it for you.
-* **Theos comes with some built-in headers and libraries for your convenience.** These are maintained as Git submodules, in the `vendor` directory. In particular, the libraries are in the plain-text format `.tbd` which is supported as of Xcode 7.0. If you have your own headers and libraries, place them in the `include` and `lib` directories in the root of Theos, not in `vendor`. You can override files in the vendor directories by using the same file path and name.
-* **Theos requires the installed copy of `dpkg-deb` to support LZMA compression.** This provides the best available compression for the package’s data. If you get an error about this flag, you need to [[update dpkg-deb|Installation#prerequisites]].
+* **Theos comes with some built-in headers and libraries for your convenience.** These are maintained as Git submodules in the `vendor` directory. In particular, the libraries are in the plain-text format `.tbd` which is supported as of Xcode 7.0. If you have your own headers and libraries, place them in the `include` and `lib` directories in the root of Theos, not in `vendor`. You can override files in the vendor directories by using the same file path and name.
+* **Theos no longer relies on `dpkg-deb`.** After lzma compression with `dpkg-deb` was deprecated, [dm.pl](https://github.com/theos/dm.pl) became Theos' [primary means of deb creation](https://github.com/theos/theos/commit/bad135c47c1c3e0300fe823e64862f490830eee3), enabling the continued use of lzma compression.
```

Comment:
Could also add a mention that dm.pl eliminates the need for an additional dependency not available on all platforms. (Specifically I believe it was originally written to fill the gap of no dpkg package existing on Cygwin)